src/modules/app/fetchMoreData.js

```diff
@@ -1,40 +0,0 @@
-// @flow
```

Comment:
:+1:

Reply:
We have now the same actions for the first and subsequent fetches, right?

Reply:
Right :+1: